holidays/countries/france.py

```diff
@@ -49,99 +49,129 @@ class France(HolidayBase):
         "Wallis-et-Futuna",
     ]
 
-    def __init__(self, **kwargs):
-        # Default subdivision to Métropole; prov for backwards compatibility
-        if not kwargs.get("subdiv", kwargs.get("prov")):
-            kwargs["subdiv"] = "Métropole"
-        HolidayBase.__init__(self, **kwargs)
+    def __init__(self, *args, **kwargs):
```

Comment:
I believe we should keep that default subdiv backward compatibility piece of code.
At least let's run it through a deprecation process steps?

Reply:
There is no specific holidays for "Métropole", It's like a synonym for "continental France". So if someone has used `holidays.FR(subdiv="Métropole")`, they will receive the same holidays set.

Reply:
Well, this makes it easier then 👍